### ISSUE TITLE
Training Siamese Network

### ISSUE THREAD

**Issue by hamzahafeez7** [Wrosinski/keraspipelines#1]
Can this Pipeline be used (or customized )to train a Siamese Network using a custom loss function.

**Comment by Wrosinski** [comment]
Hello,

The pipeline cannot be used as-is for siamese networks but it can be tuned to take as input a list of tuple of inputs for the network. 
Custom loss can also be added but then it's easier to just save/load weights from keras checkpoints instead of the full model.
